packages/twenty-shared/src/utils/image/getImageAbsoluteURI.ts

```diff
@@ -0,0 +1,29 @@
+type ImageAbsoluteURI<T extends string | null | undefined> = T extends string
+  ? string
+  : null;

```

Comment:
We talked internally, and we would like to remove this type. It will impact the function's return type.
I planned to do it myself but if you want to do it feel free :)